src/Helper/Navigation/PluginManager.php

```diff
@@ -15,15 +16,12 @@
  */
 class PluginManager extends HelperPluginManager
 {
-    /**
-     * @var string Valid instance types.
-     */
     protected $instanceOf = AbstractHelper::class;
 
     /**
      * Default aliases
      *
-     * @var string[]
+     * @inheritdoc
```

Comment:
I personally prefer `array<string,string>` than `inheritdoc` since using inheritdoc always depends on the tooling which is used around:
- Some IDEs are not able to "merge" informations
- Codestandard allows "inheritdoc" even if there is nothing to inherit (e.g. when the parent property is removed)

So this could become less accurate in upstream projects when changing to `inheritdoc`. Not sure if we should do this.

Reply:
I reverted, but I think the original reason I went with inheritdoc is because `array<string, string>` causes psalm errors - SM is annotated with `array<array-key, string>` in the version used, so I thought that inheritdoc would shortcut to the full config spec when View can catch up with more recent versions of Service Manager. Anyhow, I've changed to `@var array<string, class-string>` with a `psalm-var` of `<array-key, string> to keep psalm happy about the inherited inferred type. Hope that's the right course of action here.